fnv.go

```diff
@@ -1,8 +1,12 @@
 package bigcache
 
+import(
+	xxhash "github.com/cespare/xxhash"
+)
 // newDefaultHasher returns a new 64-bit FNV-1a Hasher which makes no memory allocations.
 // Its Sum64 method will lay the value out in big-endian byte order.
 // See https://en.wikipedia.org/wiki/Fowler–Noll–Vo_hash_function
+
```

Comment:
I've no idea how to use github but I do hope you can just "fix" or "delete" or pastebin what you think I should do next so I can be a "contributor" to this. That's all. sorry about this. I'm learning github because of bigcache.